### PR TITLE
Implement -Yno-predef and -Yno-imports

### PR DESCRIPTION
**Pull request by jvican**
The following PR addresses https://github.com/lampepfl/dotty/issues/1294 and adds three unit tests.

### CODE REVIEW

File: tests/pending/run/t7319.scala [hunk 1]
```diff
@@ -1,14 +0,0 @@
-import scala.tools.partest.ReplTest
```
Comment (smarter) [inline tests/pending/run/t7319.scala]: Did you intend to move this test to `tests/run/` instead of deleting it?

Comment (jvican) [inline tests/pending/run/t7319.scala]: Yes, sorry, Intellij played dirty on me.

File: tests/neg/t7319c.flags [hunk 2]
```diff
@@ -0,0 +1 @@
+-Yno-import
```
Comment (DarkDimius) [inline tests/neg/t7319c.flags]: Dotty ignores flag files currently. 
You'll need to create a new test similarly to how it's done here:
https://github.com/lampepfl/dotty/blob/master/test/dotc/tests.scala#L128

**Comment by jvican** [comment]
This is ready for review.

**Comment by odersky** [comment]
LGTM

**Comment by nicolasstucki** [comment]
You could change the commit message to `Fix #1294: Implement -Yno-predef and -Yno-imports` to follow the GitHub conventions.

**Pull request merged**